packages/components/tag/src/tag.ts

```diff
@@ -9,7 +9,10 @@ export const tagProps = buildProps({
     values: ['success', 'info', 'warning', 'danger', ''],
     default: '',
   },
-  hit: Boolean,
+  hit: {
+    type: Boolean,
+    default: false,
```

Comment:
the default value of boolean prop is `false`. Why should we explicitly define it?

Reply:
ok.